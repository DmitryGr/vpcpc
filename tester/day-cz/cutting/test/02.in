134
182 879
225 878
253 874
278 868
303 861
61 851
344 842
365 830
382 817
398 804
413 791
-45 777
208 768
-56 765
146 761
129 756
-69 750
452 744
457 737
324 729
466 721
55 712
352 703
211 696
235 693
163 689
-110 684
370 680
130 674
-117 668
-120 662
-122 656
306 651
198 647
236 644
171 640
498 636
262 631
149 626
143 621
328 617
230 614
-10 610
502 607
70 602
69 598
-14 595
-139 592
188 589
-16 585
-140 582
224 580
197 577
215 574
504 572
249 569
339 567
213 564
224 562
221 560
221 558
217 556
212 554
218 552
61 549
198 547
338 545
114 542
-143 539
213 537
195 534
290 531
289 528
500 526
500 522
330 518
-18 514
65 511
223 509
649 505
-141 500
69 496
71 492
-140 488
206 485
645 483
488 478
149 473
155 468
-135 463
276 459
478 455
257 451
360 448
215 445
352 440
630 434
-126 427
-124 421
131 415
140 409
302 404
28 398
612 394
259 390
212 387
607 383
418 373
597 365
400 356
388 348
88 340
365 333
113 325
125 319
140 313
308 308
173 303
-58 299
225 297
538 289
525 275
-20 260
492 247
472 233
31 220
51 208
71 198
92 188
364 180
340 173
165 167
287 163
225 160
