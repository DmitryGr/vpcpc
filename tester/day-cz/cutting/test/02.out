271 271 270 272
-310 310 -311 309
92 0 92 2
-170 169 -171 168
217 1 217 3
0 208 -2 208
554 555 553 556
0 496 -2 496
51 1 51 3
-1 427 -3 427
409 409 408 410
211 1 211 3
360 0 360 2
393 393 392 394
287 287 286 288
0 668 -2 668
0 526 -2 526
0 626 -2 626
398 0 398 2
155 1 155 3
-279 279 -280 278
612 0 612 2
0 558 -2 558
302 0 302 2
-201 200 -202 199
336 336 335 337
598 598 597 599
-56 0 -56 2
366 366 365 367
-1 319 -3 319
-140 140 -141 139
222 222 221 223
483 483 482 484
